compiler/next/include/chpl/types/BasicClassType.h

```diff
@@ -57,7 +57,12 @@ class BasicClassType final : public CompositeType {
 };
 
 
-} // end namespace uast
+} // end namespace types
+template<> struct stringify<chpl::types::BasicClassType> {
```

Comment:
I'd expect this one is not necessary because it should find the Type stringify

Reply:
I expected the same for this and others (Expression, for example), but when I remove them I get compilation errors like below:
```
error: initializer specified for non-virtual method ‘std::string chpl::stringify<T>::operator()(chpl::StringifyKind, const T&) const [with T = chpl::types::BasicClassType; std::string = std::__cxx11::basic_string<char>]’
  56 |   std::string operator()(StringifyKind stringKind, const T& stringMe) const = 0;
```
Which I am interpreting to mean that the specialization for the type `chpl::types::BasicClassType` wasn't found and the compiler couldn't, or didn't, use the one for `Type`. 
My concern here is that we're both expecting the same, different behavior than what I'm seeing.

Reply:
Yeah. I'd suppose that the problem is that the Type one isn't visible at the point that error comes up, for some reason.